client/src/Pos/Client/KeyStorage.hs

```diff
@@ -3,120 +3,139 @@
 {-# LANGUAGE TypeFamilies #-}
 
 module Pos.Client.KeyStorage
-       ( MonadKeysRead (..)
-       , MonadKeys (..)
-       , getSecretDefault
-       , modifySecretPureDefault
-       , modifySecretDefault
-
-       , getPrimaryKey
-       , getSecretKeys
-       , getSecretKeysPlain
-       , addSecretKey
-       , deleteAllSecretKeys
-       , deleteSecretKeyBy
-       , newSecretKey
-       , KeyData
-       , KeyError (..)
-       , AllUserSecrets (..)
-       , keyDataFromFile
-       ) where
+    ( MonadKeysRead (..)
+    , MonadKeys (..)
+    , AllUserSecrets (..)
+    , AllUserPublics (..)
+    , getSecretDefault
+    , getPublicDefault
+    , modifySecretPureDefault
+    , modifyPublicPureDefault
+    , modifySecretDefault
+    , modifyPublicDefault
+    , getSecretKeys
+    , getPublicKeys
+    , getSecretKeysPlain
+    , getPublicKeysPlain
+    , addSecretKey
+    , addPublicKey
+    , deleteAllSecretKeys
+    , deleteAllPublicKeys
+    , deleteSecretKeyBy
+    , deletePublicKeyBy
+    ) where
```

Comment:
This module provides an ability to work with secret and public keys:

1. Secret keys are for regular (internal) wallets. 
2. (Extended) Public keys are for external wallets.

Secret keys are stored in the local file (by default is `secret.key`). Public keys are stored in the local file as well (by default is `public.key`).